validphys2/src/validphys/plotoptions/core.py

```diff
@@ -311,10 +311,7 @@ def get_xq2map(kintable, info):
     """
     try:
         return info.process_type.xq2map(kintable, info.ds_metadata)
```

Comment:
Unrelated but stumbled upon it during CR: the return within `try` statements is not recommended by [PEP 8](https://peps.python.org/pep-0008/#programming-recommendations).

Reply:
I can modify it here if you want. While my hope is to remove this whole module, as long as it doesn't really mess up with other parts of the code (the kinematics override was messing up) it will take me a long time to actually get to it.

Reply:
On second thoughts, this function is no longer needed so I'll remove it.

Reply:
Perfect! Thanks.